src/condor_schedd.V6/schedd.cpp

```diff
@@ -1380,6 +1381,49 @@ Scheduler::updateSubmitterAd(SubmitterData &SubDat, ClassAd &pAd, DCCollector *c
 }
 
 
+int ScheddStatistics::CountJobsTick(time_t current_time, int & advance_days)
+{
+	const int seconds_per_bucket = schedd_daily_usage_stats::hour_bucket_sec;
+
+	advance_days = 0;
+
+	// how much time since the last tick? if 0 then we have nothing to do
+	// if PrevCountJobsTime nonzero and invalid, assume advancing by 365 days will clear everything.
```

Comment:
This comment seems inaccurate from code... states 365 day advancement but we only advance by 30 days